app/javascript/dashboard/components-next/avatar/Avatar.vue

```diff
@@ -69,7 +78,7 @@ const AVATAR_COLORS = {
 const STATUS_CLASSES = {
   online: 'bg-n-teal-10',
   busy: 'bg-n-amber-10',
-  offline: 'bg-n-slate-10',
+  ...(props.hideOfflineStatus ? {} : { offline: 'bg-n-slate-10' }),
```

Comment:
Can you check this is updating properly or not. The `STATUS_CLASSES` is not reactive, changes to the prop won't update the STATUS_CLASSES object?